test_3.py

```
@@ -0,0 +1 @@
+asdf
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)